src/app/public/modules/split-view/split-view-media-query.service.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  SkyMediaBreakpoints,
+  SkyMediaQueryListener
+} from '@skyux/core';
+
+import {
+  BehaviorSubject
+} from 'rxjs/BehaviorSubject';
+
+import {
+  Subscription
+} from 'rxjs/Subscription';
+
+@Injectable()
+export class SkySplitViewMediaQueryService {
```

Comment:
This is nearly a copy/paste from Flyout. Just a thought: could we put this file in a universal location and have both libraries use it?
https://github.com/blackbaud/skyux-flyout/blob/master/src/app/public/modules/flyout/flyout-media-query.service.ts

Reply:
I had the same thought... Should it be part of core's `SkyMediaQueryService`?

Reply:
We can chat about this in the morning, with the three of us.

Reply:
Let's hold off on any decisions for now.